### PR TITLE
signup_ctrl converted to es6 

### PR DESCRIPTION
**Pull request by karinashenaz**

### CODE REVIEW

File: assets/js/controllers/settings/signup_ctrl.js [hunk 1]
```diff
@@ -0,0 +1,173 @@
...
+  $scope.currentStep = 1;
+  $scope.working = false;
+  $scope.languages = Wallet.languages;
+  $scope.currencies = Wallet.currencies;
+  $scope.alerts = Wallet.alerts;
+  $scope.status = Wallet.status;
+  $scope.$watch("status.isLoggedIn", function(newValue) {
+    if (newValue) {
+      $scope.busy = false;
+      return $state.go("signup.finish.show");
```
Comment (Sjors) [inline assets/js/controllers/settings/signup_ctrl.js]: @karinashenaz return statement not needed

**Comment by Sjors** [comment]
Can you add line-breaks between the functions?

File: assets/js/controllers/settings/signup_ctrl.js [hunk 2]
```diff
@@ -0,0 +1,173 @@
...
+    if ($scope.isValid[0]) {
+      return $scope.nextStep();
+    }
+  };
+  $scope.nextStep = function() {
+    $scope.validate();
+    if ($scope.isValid[$scope.currentStep - 1]) {
+      if ($scope.currentStep === 1) {
+        $scope.working = true;
+        return $scope.createWallet(function(uid) {
```
Comment (Sjors) [inline assets/js/controllers/settings/signup_ctrl.js]: return not needed

File: assets/js/controllers/settings/signup_ctrl.js [hunk 3]
```diff
@@ -0,0 +1,173 @@
...
+      windowClass: "bc-modal terms-modal"
+    });
+  };
+  $scope.close = function() {
+    Wallet.clearAlerts();
+    return $state.go("wallet.common.home");
+  };
+  $scope.tryNextStep = function() {
+    if ($scope.isValid[0]) {
+      return $scope.nextStep();
```
Comment (Sjors) [inline assets/js/controllers/settings/signup_ctrl.js]: return not needed

Comment (karinashenaz) [inline assets/js/controllers/settings/signup_ctrl.js]: fixed

**Pull request merged**